### ISSUE TITLE
Update the wire page headings for a11y navigation

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#2454]
During a11y testing the user suggested we update the heading structure of the wire page to allow for easier navigation wire posts on this page. There should be an h2 for the wire post lists and each post should contain or be wrapped in an h3. The only h2 before the posts is a hidden h2 to navigate to the site menu. Perhaps look how twitter formats lists of tweets. Screen reader only style class `wb-inv`.

![Image](https://user-images.githubusercontent.com/7603237/186941475-981d9035-8da9-4ff1-9221-aa3d501969d6.PNG)
